README.md

```diff
@@ -308,7 +308,7 @@ Frequent Problems
   git it looks like you have deleted all files, when in fact they have
   never been checked out. Just do a checkout of the branch you want.
 
-* Receive `Error: repository has at least one unnamed head: hg r491`
+* `Error: repository has at least one unnamed head: hg r491`
```

Comment:
Perhaps say `Error: repository has at least one unnamed head: hg r<N>` to make it clear that the actual number isn't important.

Reply:
Done...